src/applications/disability-benefits/526EZ/tests/helpers.unit.spec.jsx

```diff
@@ -316,4 +380,144 @@ describe('526 helpers', () => {
       });
     });
   });
+
+  describe('hasGuardOrReservePeriod', () => {
+    it('should return true when reserve or guard period present', () => {
```

Comment:
Should we split these tests out? The way it is now, it only tests the reserve bit, not the guard. It's pretty duplicative, but maybe worthwhile?

Reply:
🤔 I mean, I don't see the harm...might as well